check-log/check-log.go

```diff
@@ -361,8 +361,8 @@ func writeFileAtomically(f string, contents []byte) error {
 		return err
 	}
 	defer os.Remove(tmpf.Name())
-	defer tmpf.Close()
 	_, err = tmpf.Write(contents)
+	tmpf.Close() // should be called before rename
```

Comment:
:bow:

@itchyny  Could you please write some helpful comment on L363 ?
I saw L363 and then tmpf.Close() can be called by defer too, but L363 doesn't work (or returns error) if rename is done correctly.

Or, I'll create another pull-request if you want.